lib/src/jira/api.rs

```diff
@@ -40,6 +40,13 @@ pub trait Session: Send + Sync {
         &self,
         proj: &str,
     ) -> Result<HashMap<String, Vec<version::Version>>>;
+
+    async fn set_release_note_text(&self, key: &str, text: &str) -> Result<()>;
+    async fn get_release_note_text(&self, key: &str) -> Result<Option<String>>;
+    async fn set_release_note_channels(&self, key: &str, channels: &str) -> Result<()>;
+    async fn get_release_note_channels(&self, key: &str) -> Result<Option<String>>;
+    async fn set_release_note_status(&self, key: &str, status: &str) -> Result<()>;
+    async fn get_release_note_status(&self, key: &str) -> Result<Option<String>>;
```

Comment:
```suggestion
    async fn set_release_note_text(&self, key: &str, text: &str) -> Result<()>;
    async fn set_release_note_channels(&self, key: &str, channels: &str) -> Result<()>;
    async fn set_release_note_status(&self, key: &str, status: &str) -> Result<()>;
 ```

Don't see these used anywhere except test mocks that expect them to never be called?